scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -704,7 +704,7 @@ class FormatOps(val tree: Tree, val initStyle: ScalafmtConfig) {
   private final def vAlignDepthUnCached(tree: Tree): Int = {
     val count: Int = tree match {
       // infix applications don't count towards the length, context #531
-      case _: Term.ApplyInfix => 0
+      case _ @(_: Term.Apply | _: Term.ApplyInfix | _: Term.Select) => 0
```

Comment:
Without this change, line2 and line3 of the below code [won't be in same block](https://github.com/scalameta/scalafmt/blob/38de09b9e7b32c8bc5b2e658b721e2d64d96570b/scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala#L333), because `vAlignDepth` of line2 and line3 are not equal, and `columnsMatch` would return 0
https://github.com/scalameta/scalafmt/blob/38de09b9e7b32c8bc5b2e658b721e2d64d96570b/scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala#L286

```scala
foo
  .x() // comment1
  .xx() // comment2
```